services/ble/Gap.proto

```diff
@@ -31,6 +50,26 @@ message SecurityLevel
     SecurityLevelEnum level = 1;
 }
 
+message SecurityModeAndLevel
+{
+    enum SecurityLevelEnum
+    {
+        None = 0;
```

Comment:
I see different casing styles used for enums. The coding standard mentions enum values should use camelCase.

Reply:
Ignoring for now.

Reply:
@EkelmansPh 
Correct, it is inconsistent even in this file.
I am not willing to alter the enums in the messages used by old central since I want to avoid updating it. 
But we can be consistent among the rest of the fields.